### PR TITLE
Changes to data management docs

### PR DESCRIPTION
**Pull request by CarrieBrown**
Merged the Storage Solutions on OSG and Guidelines for Data Management into a single document and made some minor changes to "Data Transfer with Globus".

### CODE REVIEW

File: start/data/data-transfer-with-globus.md [hunk 1]
```diff
@@ -1,20 +1,19 @@
...
+## Overview
 
 Use Globus for transferring data to and from OSG Connect and its storage
 service [Stash](<http://stash.osgconnect.net/>).  For data to be used on the
 OSG, move the data into your storage area on Stash. From the OSG Connect login
 host, `login.osgconnect.net`, this appears as: `/stash/user/your-username`.
 
-Transfer data between existing servers
---------------------------------------
+## Transfer data between existing servers
```
Comment (ChristinaLK) [inline start/data/data-transfer-with-globus.md]: maybe "transfer data between existing Globus endpoints"?

File: start/data/data-transfer-with-globus.md [hunk 2]
```diff
@@ -28,28 +27,24 @@ Transfer data between existing servers
     the View Transfer window to see your transfer progress.  Click on the
     progress bar to drill in on transfer details.
 
-Transferring between your laptop and OSG Connect Stash
-------------------------------------------------------
+## Transferring between your laptop and OSG Connect Stash
 
 To add your own laptop as an endpoint, so that you can transfer data to/from
 stash, you need to install Globus Connect, by clicking on the "Get Globus
 Connect" in the upper right corner of the Start Transfer page. More detailed
```
Comment (ChristinaLK) [inline start/data/data-transfer-with-globus.md]: repeat instructions for accessing the "Start Transfer" page, or reference the instructions above?

File: start/data/data-transfer-with-globus.md [hunk 3]
```diff
@@ -28,28 +27,24 @@ Transfer data between existing servers
...
 instructions for installing Globus Connect can be found
 here: <https://www.globusonline.org/globus_connect/>.
 
- 
 
-For servers
------------
+## For servers
```
Comment (ChristinaLK) [inline start/data/data-transfer-with-globus.md]: "Adding a server as a Globus endpoint"

File: start/data/osgconnect-storage.md [hunk 4]
```diff
@@ -12,21 +12,65 @@
 
 Here, the `username` is your login name.. For convenience, the stash and public directories are accessed from home. Here, the `username` is your login name.
```
Comment (ChristinaLK) [inline start/data/osgconnect-storage.md]: Repeated sentence: old typo?

File: start/data/osgconnect-storage.md [hunk 5]
```diff
@@ -12,21 +12,65 @@
...
 
-## home
-Home is meant for storing files long-term. Usually, files such as source code, parameter files, scripts, etc. are kept in your `/home` directory. The disk quota on home is 20 GBs. When a user exceeds his quota, the system will send email notifications. Eventually the user will loose the privilege to write on his home.
+## Storage Options Quick Reference
+
+### Data storage options
+
+|   | **Default Limit**| **Purpose** | **Network mounted** | **Backed Up** | **Quota** | **Purge** |
+|:------- |:----------------:|:------|:------:|:------:|:------:|:------:|:----------|
+| **home**    |  < 20 GB      | Meant for quick data access and not for submitting jobs.| Yes | Yes | 20 GB | No |
```
Comment (ChristinaLK) [inline start/data/osgconnect-storage.md]: What does "quick data access" mean here?

File: start/data/osgconnect-storage.md [hunk 6]
```diff
@@ -12,21 +12,65 @@
...
+| **local-scratch**   |  < 25 GB      | Meant for large temporary storage and I/O for your jobs. Files older than 30 days are automatically removed. | No | No | No | Yes, 30 days |
+| **stash**   |  < 200 GB      | Meant for large storage and I/O for your jobs. | Yes | No | No | No |
+| **public**  |  < 10 GB    | Meant for sharing data and transfer input data via HTTP or staschcp | Yes | No | No | No |
```
Comment (ChristinaLK) [inline start/data/osgconnect-storage.md]: This is *also* meant for I/O for jobs.

File: start/data/osgconnect-storage.md [hunk 7]
```diff
@@ -12,21 +12,65 @@
...
+### stash
+Stash provides medium term storage (less 6 months) for users. Stash is a distributed shared filesystem mounted on all our submit nodes, and is therefore slower than local-scratch. There is no disk quota imposed, nor a automatic purge, on stash. However, the data on stash is not backed up, so you should regularly transfer your data on stash to a long term archival system.  If you are transfering a large amount of data, please use the Globus transfer service.  For more details check the articles on [data transfer techniques](https://opensciencegrid.freshdesk.com/a/solutions/folders/12000013267).
```
Comment (ChristinaLK) [inline start/data/osgconnect-storage.md]: (less 6 months) --> (less than 6 months)

This isn't actually true, so I'm not sure it should go in the guide until we're going to enforce it.

File: start/data/osgconnect-storage.md (hunk 4 above)
Comment (CarrieBrown) [reply start/data/osgconnect-storage.md]: Copy/paste error. Oops!

**Comment by CarrieBrown** [comment]
@ChristinaLK I think I addressed all of the items you pointed out and made a few other small edits. Mind taking another look?

**Comment by ChristinaLK** [comment]
Looks good to me for now.

**Pull request merged**